src/style.css

```diff
@@ -0,0 +1,31 @@
+body {
+  background-color: #000;
+  padding: 0 12px;
+}
+
+.first,
+.third,
+.second {
+  /* box-sizing: border-box; */
+  background-color: #ddd;
+  border: 1px solid #e95d49;
+  padding: 10px;
+  margin: 0;
+}
```

Comment:
First - don't leave commented code.
Second - you don't need to create three separate classes here, it can be just one class.